nixos/modules/services/cluster/kubernetes/apiserver.nix

```diff
@@ -272,11 +272,30 @@ in
   ###### implementation
   config = mkMerge [
 
-    (mkIf cfg.enable {
+    (mkIf cfg.enable (let
+      apiserverPaths = [
```

Comment:
I fear there might be an issue regarding compatibility with `easyCerts = false` here? It should be allowed to run a less secure cluster with `AlwaysAllow` as authorizer.  I this we should consider putting some of these path-lists in pki.nix, to make it possible to opt-out of some of these path dependencies.

Reply:
Yes, i understand. Will move all occurrences into pki.nix

Reply:
Resolved in https://github.com/NixOS/nixpkgs/pull/56789/commits/7323b77435f69362b0b4cc7edcb0915e9ab1ff48#diff-8d468cb36655335308a8eb4c3999b59aR372

Reply:
Moved the remaining path units and usage in https://github.com/NixOS/nixpkgs/pull/56789/commits/ff382c18c8f8e3eba1fc3ff331b7146bcb3af674

Reply:
Looks good. Will have to test without easyCerts to see whether everything is still compatible.